fhir-server-test/src/test/java/com/ibm/fhir/server/test/operation/EverythingOperationTenant1Test.java

```diff
@@ -0,0 +1,290 @@
+            // All the possible required operations
+            Collection<FHIRPathNode> tmpResults = evaluator.evaluate(evaluationContext, EXPRESSION_OPERATION);
+            Collection<String> listOfOperations = tmpResults.stream().map(x -> x.getValue().asStringValue().string()).collect(Collectors.toList());
+
+            if (!listOfOperations.contains("everything")) {
+                LOG.warning("Operation $everything not found on server, Skipping integration test for $everything");
+                SKIP = true;
+            } else {
+                SKIP = false;
+            }
+        }
+    }
+
+    /**
+     * Create a Bundle of 5 resources of various kinds representing a patient's history and save the
+     * resource types and IDs of the created resources to eventually ensure that all resources are included
+     * in an $everything invocation.
+     *
+     * @throws Exception
+     */
+    @Test(groups = { "fhir-operation" })
+    public void testCreatePatientWithEverything() throws Exception {
+        if (SKIP) {
+            return;
+        }
+        Bundle patientBundle = TestUtil.readLocalResource("everything-operation/Patient999.json");
+        Entity<Bundle> entity = Entity.entity(patientBundle, FHIRMediaType.APPLICATION_FHIR_JSON);
+
+        Response response = getWebTarget().request().header("X-FHIR-TENANT-ID", "tenant1").header("X-FHIR-DSID", "profile").post(entity, Response.class);
+
+        assertResponse(response, Response.Status.OK.getStatusCode());
+        Bundle responseBundle = response.readEntity(Bundle.class);
+        assertFalse(responseBundle.getEntry().isEmpty());
+        for (Entry entry : responseBundle.getEntry()) {
+            com.ibm.fhir.model.resource.Bundle.Entry.Response transactionResponse = entry.getResponse();
+            assertEquals(transactionResponse.getStatus().getValue(), Integer.toString(Response.Status.CREATED.getStatusCode()));
+            String[] locationElements = transactionResponse.getLocation().getValue().split("/");
+            assertTrue(locationElements.length > 2, "Incorrect location URI format: " + transactionResponse.getLocation());
+            String resourceType = locationElements[0];
+            String resourceId = locationElements[1];
+            List<String> resources = createdResources.computeIfAbsent(resourceType, k -> new ArrayList<>());
+            resources.add(resourceId);
+        }
+    }
+
+    @Test(groups = { "fhir-operation" }, dependsOnMethods = { "testCreatePatientWithEverything" })
+    public void testPatientEverything() {
+        if (SKIP) {
+            return;
+        }
+        // Get the patient ID and invoke the $everything operation on it
+        patientId = createdResources.get("Patient").get(0);
+        Response response = getWebTarget().path("Patient/" + patientId + "/$everything").request().header("X-FHIR-TENANT-ID", "tenant1").header("X-FHIR-DSID", "profile").get(Response.class);
```

Comment:
minor formatting
```suggestion
        Response response = getWebTarget().path("Patient/" + patientId + "/$everything").request()
                .header("X-FHIR-TENANT-ID", "tenant1")
                .header("X-FHIR-DSID", "profile")
                .get(Response.class);
```

Reply:
and the same applies to all of the long JAX-RS client lines in this test

Reply:
Done